bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/ferroamp.xml

```diff
@@ -0,0 +1,449 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="energyhub">
```

Comment:
Both ferroamp.xml and thing-type.xml have the same `energyhub` thing id. Please remove one.

Reply:
20241024
Deleted ferroamp.xml and channels.xml and pasted this info. to thing-types.xml.
I saw that almost every binding has it like that, and I think it also might be that the binding starts a little bit smother this way.